### DISCUSSION TITLE
solution/20230418C_C/

### DISCUSSION THREAD

**Discussion by giscus** [X-six-floor-slip-liu-X/X-six-floor-slip-liu-X.github.io#4] [Announcements]
# solution/20230418C_C/

这是一个中学 OIer 的博客，他会在这里分享一些算法和题解，喜欢可以进来贴贴。

https://x-six-floor-slip-liu-x.github.io/solution/20230418C_C/

**Comment by Dreamerkk** [comment]
好神仙的题。


~~为什么part1过了就变成part3了？~~

**Reply by X-six-floor-slip-liu-X** [reply]
草，我改改
